cypress/e2e/tests/migration/migration-waves/export_to_jira_cloud.test.ts

```diff
@@ -74,7 +76,7 @@ describe(["@tier1"], "Export Migration Wave to Jira Cloud", function () {
     Object.values(JiraIssueTypes).forEach((issueType) => {
         it(`Export wave as ${issueType} to Jira`, function () {
             jiraCloudInstance
-                .getProject()
+                .getProject(Cypress.env("jira_atlassian_cloud_project"))
```

Comment:
And what if you will need to use Server stage account?

Reply:
That goes into a different file, this file is only for export to jira cloud